### PR TITLE
Add the firstLogicalProcessor controlling interface

### PR DESCRIPTION
**Pull request by jsunintel**

### CODE REVIEW

**Comment by jsunintel** [comment]
@tianjunwork Please kindly review.

File: Source/API/EbApi.h [hunk 1]
```diff
@@ -19,7 +19,7 @@ extern "C" {
 #define EB_HME_SEARCH_AREA_ROW_MAX_COUNT        2
 
 #define EB_THREAD_COUNT_MIN_CORE               48
-#define EB_THREAD_COUNT_FACTOR                  2
+#define EB_THREAD_COUNT_FACTOR                  1
```
[the remark this answers is not in the document]
Comment (tianjunwork) [reply Source/API/EbApi.h:22]: @jsunintel, what is the impact on performance?

[the remark this answers is not in the document]
Comment (jsunintel) [reply Source/API/EbApi.h:22]: > 

No impact on performance by default. It doesn't change the default value, but the allowed min value

File: Source/Lib/Codec/EbEncHandle.c [hunk 2]
```diff
@@ -1717,7 +1727,7 @@ void LoadDefaultBufferConfigurationSettings(
     if (sequenceControlSetPtr->staticConfig.threadCount != 0)
         totalThreadCount = sequenceControlSetPtr->staticConfig.threadCount;
     else
-        totalThreadCount = coreCount * EB_THREAD_COUNT_FACTOR;
+        totalThreadCount = coreCount * numGroups;
```
[the remark this answers is not in the document]
Comment (tianjunwork) [reply Source/Lib/Codec/EbEncHandle.c:1730]: @jsunintel , how about this one?

**Comment by tianjunwork** [comment]
all configuration parameters are in this table, @jsunintel could you add -flp in this table?
https://github.com/OpenVisualCloud/SVT-HEVC/blob/48e8800ade0bd1e5f4233d1ae6125d245bdb5da8/Docs/svt-hevc_encoder_user_guide.md#list-of-all-configuration-parameters

also, the detailed usage in:
https://github.com/OpenVisualCloud/SVT-HEVC/blob/48e8800ade0bd1e5f4233d1ae6125d245bdb5da8/Docs/svt-hevc_encoder_user_guide.md#2-thread-management-parameters

**Review by tianjunwork — APPROVED** [review]

**Comment by 1480c1** [comment]
Thanks @jsunintel @tianjunwork

**Pull request merged**